From: Ilya Dryomov <idryomov@gmail.com>
Date: Wed, 22 Apr 2015 18:28:13 +0300
Subject: [PATCH] rbd: rbd_wq comment is obsolete
Git-commit: f77303bddabf73ebccb60f613b77da391f933cf6
Patch-mainline: v4.1
References: Fate#318586


After the switch to blk-mq rbd_wq processes requests, not devices.

Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Acked-by: NeilBrown <neilb@suse.com>

---
 drivers/block/rbd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/block/rbd.c
+++ b/drivers/block/rbd.c
@@ -6226,7 +6226,7 @@ static int __init rbd_init(void)
 
 	/*
 	 * The number of active work items is limited by the number of
-	 * rbd devices, so leave @max_active at default.
+	 * rbd devices * queue depth, so leave @max_active at default.
 	 */
 	rbd_wq = alloc_workqueue(RBD_DRV_NAME, WQ_MEM_RECLAIM, 0);
 	if (!rbd_wq) {
